wiki/Tournaments/ANZT/2018_Summer/en.md

```diff
@@ -1,36 +1,37 @@
-Aus/NZ osu!standard Tournament 5 - Summer
-=============================
+# Aus/NZ osu!standard Tournament 5 - Summer
 
-![Osu! AU/NZ Tournament 5 Summer](logo.jpg)
+*See also: [Aus/NZ osu!standard Tournament 5 - Winter](/wiki/Tournaments/ANZT/2018_Winter)*
```

Comment:
are these two Summer/Winter ANZTs any more related to each other than the rest of the ANZT series (other than the number)? if not, I don't think you should include this "See also" note